website/content/en/blog/the-economics-of-the-cncf-platform-engineering-model.md

```diff
@@ -0,0 +1,84 @@
+
+DevOps platforms or Internal Developer Platforms (IDP) operated by platform engineering teams have many characteristics of multi-sided markets. For a software developer there is no reason to join an IDP if it does not improve the developer experience beating the experienced cost of switching. Similarly, for different [platform teams](https://teamtopologies.com/key-concepts) in the company there is no reason to invest into an IDP without having developers using it. This shows one important aspect of multi-sided markets. Adoption looks like a chicken and egg problem. Before discussing adoption in detail, let’s speak a bit about the many sides of a DevOps platform.
+
+I called these platforms many sided. The [CNCF Platform White Paper](https://tag-app-delivery.cncf.io/whitepapers/platforms/#why-platforms) describes the motivations why enterprises might invest into platforms:
+
+“The desire to refocus delivery teams on their core focus and reduce duplication of effort across the organisation has motivated enterprises to implement platforms for cloud-native computing. By investing in platforms, enterprises can:
+
+1. Reduce the cognitive load on product teams and thereby accelerate product development and delivery
+2. Improve reliability and resiliency of products relying on platform capabilities by dedicating experts to configure and manage them
+3. Accelerate product development and delivery by reusing and sharing platform tools and knowledge across many teams in an enterprise
+4. Reduce risk of security, regulatory and functional issues in products and services by governing platform capabilities and the users, tools and processes surrounding them
+5. Enable cost-effective and productive use of services from public clouds and other managed offerings by enabling delegation of implementations to those providers while maintaining control over user experience”
+
+By analysing this list, we can see the interest of different actors within the enterprise. Reducing the cognitive load of product teams is the direct interest of the members of the product team and the indirect interest of the company as it can accelerate product development. Improving reliability and resilience is the interest of infrastructure teams. Reducing security risks and increasing compliance is the interest of security teams. While cost considerations are a financial topic and depending on the company culture might live with the higher management only. These are the different sides of the market. A successful platform needs to balance the needs of all the sides and orchestrate adoption across all of them.
+
+## Adopting a platform
+
+As already noted, a central question of multi-sided markets is their adoption. The more users are on a platform, the more investment goes into a platform. The more features a platform offers, the more users want to use a platform. But how does one start this cycle? Where is the turning point that will start a self enforcing spiral of adoption?
```

Comment:
This sounds more to me like product-market fit, common to all products. Again, I'm not grokking why this is specific to a 2-sided market.

Reply:
@earth2marsh Product-market fit is about feature adoption, not platform adoption. AFAIK.

Reply:
@earth2marsh @abangser I might be able to better explain this through Customer Lifetime Value. Let's assume that our "platform" has a market of fully identical users. The first user joining bring CLV €100. 

On a "regular" market, the 2nd user joining will have a CLV €100 as well, and the CLV of the first user does not change.

On a networked market (and every two-sided market has network effect, but not every networked market is two-sided), the CLV of the first user has two components:

1. their own CLV
2. the value their presence brings to the following users

Thus the 2nd users will attribute higher than €100 to the platform, even though their CLV is €100 (+ the value their presence brings to the 3rd+ users)

In the Internal Dev Platform area, as argued in this article, there are multiple sides to the platform. The platform in itself is just a skeleton, a service registry, a scaffolding engine.

1. a platform team starts to develop templated for the platform
   - this brings value to developer users
2. some developers join the platform
   - this brings value to the already present platform team AND
   - this brings value - for example - to the security team
3. the security platform team joins the platform to add their own tools
4. ...

Does this help to understand the two-sided nature better?